src/utils/country.utils.ts

```diff
@@ -0,0 +1,18 @@
+import Cookies from "js-cookie";
+import { cloudflareTrace } from "../constants/url.constants";
+
+interface TraceData {
```

Comment:
```suggestion
type TraceData = {
```

Just for consistency